Lib/ufo2ft/featureWriters/cursFeatureWriter.py

```diff
@@ -97,9 +99,9 @@ def _makeCursiveStatements(self, glyphs):
             for anchor in glyph.anchors:
                 if entryAnchor and exitAnchor:
                     break
-                if anchor.name == "entry":
+                if anchor.name == self.entry:
```

Comment:
if you use the options dict like I suggested above, you can then use these as `self.options.entry` or `self.options.exit` etc.